static/src/stylesheets/module/commercial/creatives/_fabric-video.scss

```diff
@@ -2,27 +2,34 @@
     position: relative;
     height: 250px;
 
-    > .creative__video,
-    > .creative__link {
+    > .creative__link,
+    .creative__video,
+    .creative__poster,
+    .creative__alt,
+    .creative__layer,
+    .creative__slide {
         position: absolute;
         top: 0;
         left: 0;
         width: 100%;
         height: 100%;
     }
 
-    > .creative__video {
+    .creative__video {
         object-fit: cover;
     }
 
-    .creative__alt,
-    .creative__layer,
-    .creative__slide {
-        position: absolute;
-        top: 0;
-        left: 0;
-        width: 100%;
-        height: 100%;
+    .creative__video--left {
+        object-position: left;
```

Comment:
Alas, nope, not even `object-fit`, which means the embedded element will probably get resized on IE.